pdk/1.0/modules/ROOT/pages/policies-pdk-troubleshooting.adoc

```diff
@@ -4,54 +4,115 @@ include::_attributes.adoc[]
+[...]
+----
+
+To trouble shoot this error:
+
+. Ensure Xcode is installed.
++
+To install Xcode, execute the following command:
++
+[source,ssh]
+----
+xcode-select --install
+----
 
-[source,shell]
+. If Xcode is installed, create or edit the `~/.cargo/config.toml` by adding the following:
++
+[source,toml]
+----
+[target.x86_64-apple-darwin]
+rustflags = [
+  "-C", "link-arg=-undefined",
+  "-C", "link-arg=dynamic_lookup",
+]
+[target.aarch64-apple-darwin]
+rustflags = [
+  "-C", "link-arg=-undefined",
+  "-C", "link-arg=dynamic_lookup",
+]
 ----
-cargo +nightly install cargo-anypoint@1.0.0-beta.1 --registry anypoint -Z registry-auth --config .cargo/config.toml
-warning: flag `-Z registry-auth` has been stabilized in the 1.74 release, and is no longer necessary
-  Authenticated registries are available if a credential provider is configured.
 
-    Updating `anypoint` index
-error: failed to download `cargo-anypoint v1.0.0-beta.1 (registry `anypoint`)`
+[[cli-credential]]
+== Anypoint CLI Credential Error
 
-Caused by:
-  unable to get packages from source
+Anypoint CLI requires Multi-factor authentication (MFA) with a Connected App. Using any PDK command with out Connected App credentials throws an error similar to the following:
 
-Caused by:
-  authenticated registries require a credential-provider to be configured
-  see https://doc.rust-lang.org/cargo/reference/registry-authentication.html for details
-make: *** [install-cargo-anypoint] Error 101
+[source,ssh]
 ----
+Error: Failed to launch the browser process! undefined
+[...]:ERROR:ozone_platform_x11.cc(239)] Missing X server or $DISPLAY
+[21921:21921:1103/113335.217708:ERROR:env.cc(255)] The platform failed to initialize. Exiting.
+TROUBLESHOOTING: https://pptr.dev/troubleshooting
+----
+
+To configure Anypoint CLI with MFA authentication using a Connected App, refer to xref:4.x@anypoint-cli::auth.adoc[].
 
-To fix the error:
+[[group-id-error]]
+== Group ID Error When Creating the Policy Project
 
-. Find your Cargo main directory (`.cargo/`).
-. In `.cargo/`, find the `credentials.toml` file and add the following new line `credential-provider = ["cargo:token"]` to the '[registries.anypoint]` section, for example:
+Anypoint CLI uses the `ANYPOINT_ORG` environment variable. This variable can use either the Organization Name or the Organization ID as the value. If the variable is set to the Organization Name, the xref:policies-pdk-create-project.adoc[policy creation command] fails to infer the Group ID of the organization. 
+
+It is recommended to use the Organization ID for the value `ANYPOINT_ORG`. However, if you cannot change the variable, there are two options to provide the Group ID:
+
+* If you run the create command and receive the following prompt:
 +
-[source,toml]
+[source,ssh]
 ----
-[registries.anypoint]
-token = <someToken>
-credential-provider = ["cargo:token"]
+Please provide a valid group-id (the id of the organization that will own the asset):
```

Comment:
Suggest to dev to use the same language as the docs. And use "ID" instead of "id".